sdk/go/docker/customTypes.go

```diff
@@ -82,7 +82,7 @@ type DockerBuildArgs struct {
 	// An optional map of named build-time argument variables to set during the Docker build.
 	// This flag allows you to pass built-time variables that can be accessed like environment variables
 	// inside the `RUN` instruction.
-	Args pulumi.MapInput `pulumi:"args"`
+	Args pulumi.StringMap `pulumi:"args"`
```

Comment:
Is this a breaking change? Users will need to update their code right?

Reply:
This is mentioned in the PR description. `Args` and `Env` don't work as implemented. There's a panic unmarshalling to the wrong type.